antarest/study/business/aggregator_management.py

```diff
@@ -41,8 +41,19 @@
 """Index in path parts starting from the Monte Carlo year to determine the Monte Carlo year."""
 AREA_OR_LINK_INDEX__IND, AREA_OR_LINK_INDEX__ALL = 2, 1
 """Indexes in path parts starting from the output root `economy//mc-(ind/all)` to determine the area/link name."""
-PRODUCTION_COLUMN_NAME = "production"
-PRODUCTION_COLUMN_REGEX = "mwh"
+RENAME_MAPPING = {
+    "mwh": "production",
+    "NP Cost - Euro": "NP Cost",
+    "NODU": "NODU",
+    "P-injection - MW": "P.injection",
+    "P-withdrawal - MW": "P.withdrawal",
+    "production": "levels",
```

Comment:
I think this doesn't work. The mapping should be "mwh": "levels" for details St storage and "mwh": "production" for other details :/, see the JIRA ticket